framework/doc/content/source/meshgenerators/CoarseElementIDFixup.md

```diff
@@ -0,0 +1,26 @@
+# CoarseElementIDFixup
```

Comment:
I do not quite like `reassignment` though because the number of elements whose ids get reassigned can be very small or possibly zero. Any better suggestions.

This generator basically ensures all elements with the same ids form a connected region through their sides. It detects the nodes that are shared by the disadjointed parts and put the elements connected with the nodes with the same element id. Cannot come up a single word for this strategy, maybe `node_based_reassignment`? Since we only have one strategy here, I do not think we need a separate parameter for choosing the strategy.

For the naming, do we need to carry `extra` all the times? On the other hand, I think we should have `element` in the name to differentiate it from `node` ids.

Reply:
regrouping, consolidation.
Should you call this ExtraElementIDConnectedFaceGroupingGenerator?
If we want a more general name, we need to have some flexibility built in. This is doing a very very precise thing, which the name doesnt give away

it s sort of node based but it s really face based. Your criterion is more on faces.

extra is a must until we find a better name. Agree with elements. Ugh so long

Reply:
How about `ExtraElementIDConnectionFixup`? I suggest remove `Generator` here because this is not generating something. I cannot come up with a better name than `Fixup`, maybe `Patch`?

Reply:
All generators are carefully renamed.